ccc/elasticsearch.py

```diff
@@ -78,6 +78,14 @@ def read_attr(name):
         meta_dict['build-name'],
     )
 
+    effective_version = os.environ.get('EFFECTIVE_VERSION')
+    if effective_version:
```

Comment:
this line can not reasonably be reached. of the env var is not set, a KeyError would have been raised